addons/UtilityClasses/ImageProcessingLibrary.cpp

```diff
@@ -4,7 +4,7 @@
  *
  * Authors:
  * 		Heikki Suhonen <heikki.suhonen@gmail.com>
- *
+ *	Array overflow fix 2017/7/18  Pete G.
```

Comment:
You wanna change that to be roughly in the same format as Haikki's entry? One more indent, name, email (if you want) and "Array overflow fix (18-07.2017)" in brackets, maybe.